src/fdosecrets/dbus/DBusClient.h

```diff
@@ -0,0 +1,152 @@
+/*
+ *  Copyright (C) 2020 Aetf <aetf@unlimitedcode.works>
+ *  Copyright (C) 2020 Jan Klötzke <jan@kloetzke.net>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef KEEPASSXC_FDOSECRETS_DBUSCLIENT_H
+#define KEEPASSXC_FDOSECRETS_DBUSCLIENT_H
+
+#include <QSet>
+#include <QSharedPointer>
+#include <QString>
+#include <QUuid>
+
+namespace FdoSecrets
+{
+    class DBusMgr;
+    class CipherPair;
+
+    enum class ItemAuth
+    {
+        Unknown,
+        Allowed,
+        AllowedOnce,
+        Denied,
+        DeniedOnce,
+    };
```

Comment:
Should standardize with the Access Control Dialog. I would just use these in the dialog context.